<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>553</width>
    <height>372</height>
   </rect>
  </property>
  <widget class="QWidget" name="centralwidget"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>553</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_File">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="action_Open"/>
    <addaction name="action_Print"/>
    <addaction name="action_Print_Strip"/>
    <addaction name="actionOutput_to_Pdf"/>
    <addaction name="separator"/>
    <addaction name="actionE_xit"/>
    <addaction name="separator"/>
   </widget>
   <widget class="QMenu" name="menu_Tools">
    <property name="title">
     <string>&amp;Tools</string>
    </property>
    <addaction name="actionBackwards"/>
    <addaction name="actionForwards"/>
   </widget>
   <widget class="QMenu" name="menu_Help">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="actionAbout"/>
    <addaction name="actionTest1"/>
    <addaction name="actionTest2"/>
    <addaction name="actionTest3"/>
   </widget>
   <widget class="QMenu" name="menu_Window">
    <property name="title">
     <string>&amp;Window</string>
    </property>
    <addaction name="action_Close"/>
    <addaction name="actionClose_All"/>
    <addaction name="separator"/>
    <addaction name="action_Tile"/>
    <addaction name="actionCascade"/>
    <addaction name="separator"/>
    <addaction name="action_Next"/>
    <addaction name="action_Previous"/>
   </widget>
   <addaction name="menu_File"/>
   <addaction name="menu_Tools"/>
   <addaction name="menu_Window"/>
   <addaction name="menu_Help"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBarFile">
   <attribute name="toolBarArea">
    <enum>RightToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="separator"/>
   <addaction name="action_Open"/>
   <addaction name="action_Print"/>
   <addaction name="action_Print_Strip"/>
   <addaction name="actionOutput_to_Pdf"/>
  </widget>
  <widget class="QToolBar" name="toolBarView">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionBackwards"/>
   <addaction name="actionForwards"/>
   <addaction name="separator"/>
  </widget>
  <action name="action_Open">
   <property name="icon">
    <iconset resource="mdi.qrc">
     <normaloff>:/out/images/open.png</normaloff>:/out/images/open.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Open</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="action_Print">
   <property name="icon">
    <iconset resource="mdi.qrc">
     <normaloff>:/out/images/print.png</normaloff>:/out/images/print.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Print</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+P</string>
   </property>
  </action>
  <action name="actionOutput_to_Pdf">
   <property name="icon">
    <iconset resource="mdi.qrc">
     <normaloff>:/out/images/printPDF.png</normaloff>:/out/images/printPDF.png</iconset>
   </property>
   <property name="text">
    <string>Output to Pdf</string>
   </property>
  </action>
  <action name="actionE_xit">
   <property name="icon">
    <iconset resource="mdi.qrc">
     <normaloff>:/out/images/Exit.png</normaloff>:/out/images/Exit.png</iconset>
   </property>
   <property name="text">
    <string>E&amp;xit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionBackwards">
   <property name="icon">
    <iconset resource="mdi.qrc">
     <normaloff>:/out/images/left2.png</normaloff>:/out/images/left2.png</iconset>
   </property>
   <property name="text">
    <string>Backwards</string>
   </property>
  </action>
  <action name="actionForwards">
   <property name="icon">
    <iconset resource="mdi.qrc">
     <normaloff>:/out/images/right2.png</normaloff>:/out/images/right2.png</iconset>
   </property>
   <property name="text">
    <string>Forwards</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionTest1">
   <property name="text">
    <string>Debug Test 1</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Alt+1</string>
   </property>
  </action>
  <action name="actionTest2">
   <property name="text">
    <string>Debug Test 2</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Alt+2</string>
   </property>
  </action>
  <action name="actionTest3">
   <property name="text">
    <string>Debug Test 3</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Alt+3</string>
   </property>
  </action>
  <action name="action_Close">
   <property name="text">
    <string>&amp;Close</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+F4</string>
   </property>
  </action>
  <action name="actionClose_All">
   <property name="text">
    <string>Close &amp;All</string>
   </property>
  </action>
  <action name="action_Tile">
   <property name="text">
    <string>&amp;Tile</string>
   </property>
  </action>
  <action name="actionCascade">
   <property name="text">
    <string>Cascade</string>
   </property>
  </action>
  <action name="action_Next">
   <property name="text">
    <string>&amp;Next</string>
   </property>
  </action>
  <action name="action_Previous">
   <property name="text">
    <string>&amp;Previous</string>
   </property>
  </action>
  <action name="action_8_Second_Strip">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;8 Second Strip</string>
   </property>
  </action>
  <action name="action_Full_Disclosure">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;Full Disclosure</string>
   </property>
  </action>
  <action name="action_Print_Strip">
   <property name="icon">
    <iconset resource="mdi.qrc">
     <normaloff>:/out/images/print_strip.png</normaloff>:/out/images/print_strip.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Print Strip</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+P</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="mdi.qrc"/>
 </resources>
 <connections/>
</ui>
